app/controllers/staff/activity_forms_controller.rb

```diff
@@ -75,6 +75,8 @@ def show
       skip_step unless @activity.is_project?
     when :sustainable_development_goals
       skip_step if @activity.fund?
+    when :gcrf_challenge_area
```

Comment:
Is this code required to make the tests in the previous commit pass? i.e. does the build pass on the previous commit or not?